stdlib/Dates/src/conversions.jl

```diff
@@ -38,26 +38,33 @@ Base.convert(::Type{Day},dt::Date) = Day(value(dt))            # Converts Date t
 
 ### External Conversions
 const UNIXEPOCH = value(DateTime(1970)) #Rata Die milliseconds for 1970-01-01T00:00:00
+const LOCALEPOCH = value(DateTime(Libc.TmStruct(0))) #Rata Die for local time at UTC 1970-01-01T00:00:00
 
 """
-    unix2datetime(x) -> DateTime
+    unix2datetime(x::Real; local_time::Bool=false) -> DateTime
```

Comment:
Maybe just `local` instead of `local_time`?   In context, it should be pretty clear that this refers to time.

Reply:
nope, you can't use that, I don't know if it's reserved or something. I tried it initially, and it didn't work, which is why I had to do this.

Reply:
Oh right, local is a reserved word.

`localtime`, then — we typically omit underscores in Julia unless readability is substantially worse.

Reply:
alright, I'll make that change.